packages/stencil/src/components/datacom-breadcrumb/usage.md

```diff
@@ -1,16 +1,23 @@
 # Breadcrumb Usage
+
 ## Web component
-Breadcrumb has one component 'datacom-breadcrumb'
-### Properties - datacom-breadcrumb
 
--url for passing link of other page. Styled with 'dc- breadcrumb' class.
+Breadcrumb has one component: 'datacom-breadcrumb'
+
+```html
+<datacom-breadcrumb url="https://datacom.com/nz/en/discover" separator>Breadcrumb</datacom-breadcrumb>
+```
+
 ```html
-   <datacom-breadcrumb url="https://datacom.com/nz/en/discover" > Breadcrumb </datcom-breadcrumb>
-    
-   ```
-   ```html 
-   <div>
-    <datacom-breadcrumb url="https://datacom.com/nz/en/discover?&page=2"  >Breadcrumb1<datacom-breadcrumb>
-     <datacom-breadcrumb >Breadcrumb2  </datacom-breadcrumb>
-   </div>
-``` 
\ No newline at end of file
+<div>
+  <datacom-breadcrumb url="https://datacom.com/nz/en/discover" separator>Breadcrumb</datacom-breadcrumb>
+  <datacom-breadcrumb url="https://datacom.com/nz/en/discover" separator>Breadcrumb</datacom-breadcrumb>
+  <datacom-breadcrumb>Breadcrumb</datacom-breadcrumb>
+</div>
+```
+### Properties - datacom-breadcrumb
+
+url-This is the url to the page where breadcrumb link redirects to.
+separator- This is prop which is used to show Icon(>) alongside breadcrumb text.
+
+"If no url is provided, the component will be rendered as plain text."
```

Comment:
You should include this line with the url prop description.

Reply:
Sorry, didn't mean to approve this yet.

Reply:
No worries, thanks for spotting it.  I fixed it.